### PR TITLE
Working search! Woohoo!

### LINKED ISSUE #26: Search - *Priority 5*
Perhaps using the work dave has done with react on Cats Gone Good

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This fixes the broken search functionality in Aurora. Awesome! Staff can now search for members by Northstar ID, Drupal ID, email, or phone (including portions of any of those fields).

Note, this relies on [some changes in Northstar](https://github.com/DoSomething/northstar/pull/257).

Closes #26.

screen shot 2016-01-21 at 5 27 57 pm

---
For review: @angaither

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -102,26 +102,26 @@ public function destroy($id)
...
+        $query = $request->query('query');
+
+        // Redirect empty queries to the user index.
+        if ($query === '') {
+            return redirect()->route('users.index');
         }
+
+        // Attempt to fetch all users.
+        $users = $this->northstar->getAllUsers(['search' => [
+            '_id' => $query,
```
Comment (angaither) [inline app/Http/Controllers/UsersController.php]: how does drupal id vs northstar id work with this?

**Comment by angaither** [comment]
okayyy

File: app/Http/Controllers/UsersController.php (hunk 1 above)
Comment (DFurnes) [inline app/Http/Controllers/UsersController.php]: Oh, it _doesn't_. Whoops. Will add `drupal_id` to the query.

**Comment by angaither** [comment]
fab

**Pull request merged**